mypy/options.py

```diff
@@ -0,0 +1,24 @@
+from mypy import defaults
+from mypy import build
+
+class Options:
+    _instance = None  # type: 'Options'
```

Comment:
As mentioned above, I'd rather not use global state. The mypy implementation is pretty careful about avoiding global mutable state, including class variables, and I'd like to keep it that way :-)